### PR TITLE
Clean rolling CI

### PR DESCRIPTION
**Pull request by ahcorde**

### CODE REVIEW

**Comment by clalancette** [comment]
The change looks OK to me, but this is failing because I think https://github.com/ros/geometry_tutorials/blob/6481e8fa9b5cef611f3d6d3aea2ea338701c481a/turtle_tf2_cpp/src/turtle_tf2_message_filter.cpp#L55 needs to be updated to pass in a QoS parameter.

**Comment by ahcorde** [comment]
@clalancette friendly ping

File: turtle_tf2_cpp/src/turtle_tf2_message_filter.cpp [hunk 1]
```diff
@@ -52,7 +52,9 @@ class PoseDrawer : public rclcpp::Node
     tf2_listener_ =
       std::make_shared<tf2_ros::TransformListener>(*tf2_buffer_);
 
-    point_sub_.subscribe(this, "/turtle3/turtle_point_stamped");
+    point_sub_.subscribe(
+      this, "/turtle3/turtle_point_stamped",
+      rclcpp::QoS(rclcpp::QoSInitialization::from_rmw(rmw_qos_profile_default)));
```
Comment (clalancette) [inline turtle_tf2_cpp/src/turtle_tf2_message_filter.cpp]: Especially given that this is one of our examples, I think this is a little wordy.  We can simplify this to:

```suggestion
    point_sub_.subscribe(this, "/turtle3/turtle_point_stamped", rclcpp::QoS(10));
```

(this probably points out a deficiency in our QoS APIs for this, but at least the above doesn't look as scary)

**Review by clalancette — CHANGES_REQUESTED** [review]

**Review by clalancette — APPROVED** [review]
Looks good to me with green CI.

**Pull request merged**